README.md

```diff
@@ -16,7 +16,14 @@ The Python data reduction pipeline for WiFeS
   - Users can create their own `.JSON` file following the same structure with their preferred setup.
 
 #### Known Problems
-- Multiprocessing not always works properly.
+- **Multiprocessing** not always works properly.
+    - **Update Dec 2023**: Multiprocessing can be enabled by **Linux users ONLY** so the pipeline will does the job faster. To enable the multithreading option, please follow these steps:
```

Comment:
Typo, and slight reword. Replace "will does" with "may do". Depending on the data & number of available cores, the single threaded version *could* be faster even on linux.